src/approaches/interactive_learning_approach.py

```diff
@@ -117,7 +121,13 @@ def _relearn_predicates_and_nsrts(
                 (self._predicates_to_learn - {pred}) | {new_pred}
 
         # Learn NSRTs via superclass
-        self._learn_nsrts(dataset.trajectories, online_learning_cycle)
+        self._learn_nsrts(self._dataset.trajectories, online_learning_cycle)
+
+        # Save the things we need other than the NSRTs, which were already
+        # saved in the above call to self._learn_nsrts()
+        save_path = utils.get_approach_save_path_str()
+        with open(f"{save_path}_{online_learning_cycle}.DATA", "wb") as f:
```

Comment:
just to check, the reason this doesn't overwrite the file that's created by the parent class is that the suffixes are different (DATA here vs NSRTs there). right?

Reply:
right